recipes/onnx/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "onnx" %}
+{% set version = "0.2" %}
+{% set sha256 = "63dde5e5f3aa88a8993c65ca35960bac06ed2233001924db28a611b7c40c6254" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - protobuf
+    - numpy
```

Comment:
Does this link against numpy? It seems like not, in which case you can drop it from `build` requirements (since you're using `--single-version-externally-managed`).

If it is linking against numpy in a way that I don't see, then you'll need to [pin the versions](https://conda-forge.org/docs/meta.html#building-against-numpy).